src/types/DaprClientOptions.ts

```diff
@@ -19,8 +19,14 @@ export type DaprClientOptions = {
    * Default is true.
    */
   isKeepAlive?: boolean;
+
   /**
    * Options related to logging.
    */
   logger?: LoggerOptions;
+
+  // The Dapr App ID
+  // This is used for proxying requests
+  // currently only supported in GRPC
+  daprAppId?: string;
```

Comment:
Is this something that we can instead use from the env var APP_ID? https://docs.dapr.io/reference/environment/

Reply:
refactored into settings as `getAppId()` it will use a default one, but take APP_ID as primary